superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -444,6 +444,18 @@ function ChartList(props: ChartListProps) {
       ),
       paginate: false,
     },
+    {
+      Header: t('Favorite'),
+      id: 'favorite',
+      col: 'id',
```

Comment:
This should be necessary, `id` should correspond to the column.

You can either add a hidden column for `id` or change the existing column from `favorite` to `id`.

Reply:
Same comment as above concerning the humanly readable filtering parameter. However, I am not sure what you are referring to by the "hidden column"? If that might solve the issue that I am proposing I'll be happy to go for it.